payments.go

```diff
@@ -143,6 +143,7 @@ func PayBlankInvoice(paymentRequest string, amountSatoshi int64) error {
 	if len(response.PaymentError) > 0 {
 		return errors.New(response.PaymentError)
 	}
+	getBackupPath()
```

Comment:
The "PayBlankInvoice" should trigger "onNewSentPayment" so we don't need to call to backup here.
I see a bug, not related to this commit, that It doesn't do it right now because there is a missing call to "syncSentPayments".
I think we should just replace the "onAccountChanged()" call to "syncSentPayments" and remove the "getBackupPath" call.

Reply:
:+1:  Great catch!